internal/controller/lease_controller.go

```diff
@@ -90,6 +91,11 @@ func (r *LeaseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 
 		// Find available exporter
 		for _, exporter := range exporters.Items {
+			if !(meta.IsStatusConditionTrue(exporter.Status.Conditions, "Registered") &&
```

Comment:
```suggestion
			if !(meta.IsStatusConditionTrue(exporter.Status.Conditions, "Registered") &&
```

We should make those constants in our v1alpha1